prow/config.yaml

```diff
@@ -814,18 +810,17 @@ postsubmits:
     - release-1.6
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:0.9
```

Comment:
Yeah I asked and no one know why this is the case. I added unit testing around this (to either ensure that the tag is current or else leave a reason for not doing so)